src/main/scala/it/pagopa/interop/backendforfrontend/api/impl/EServicesApiServiceImpl.scala

```diff
@@ -330,22 +330,24 @@ final case class EServicesApiServiceImpl(
     requesterTenant <-
       if (requesterId != eService.producerId) tenantProcessService.getTenant(requesterId)
       else Future.successful(producerTenant)
-
     activeDescriptor = getActiveDescriptor(eService)
     latestAgreement <- agreementProcessService.getLatestAgreement(requesterId, eService)
+    hasCertifiedAttributes = activeDescriptor.fold(false)(descriptor =>
```

Comment:
just to double check, is this the expected behaviour from the frontend?

Reply:
I'm pretty sure I've discussed it with @ruggerocastagnola, can you confirm?

Reply:
I just checked the logic with @ruggerocastagnola and it's ok